backend/data/annotations/tocs/iliad-folios/toc.iliad-folios.json

```diff
@@ -0,0 +1,2504 @@
+{
+  "@id": "urn:cite:scaife-viewer:toc.iliad-folio-ref-root",
+  "title": "Iliad Books by Folio Ref",
+  "description": "Mapping between book / line boundaries to Venetus A folios",
+  "items": [
+    {
+      "@id": "urn:cite:scaife-viewer:toc.iliad-folio-ref-1",
+      "title": "1",
+      "uri": "urn:cite:scaife-viewer:toc.iliad-folio-ref-1",
```

Comment:
Dropt his

Reply:
@jtauber You had asked about `uri` vs `id`; we _do_ want `uri` here because this is an _item_ and would be listed beneath the "root" of `urn:cite:scaife-viewer:toc.iliad-folio-ref-root`.

`urn:cite:scaife-viewer:toc.iliad-folio-ref-root` itself does _not_ have a `uri` field, but only `id`.